### PR TITLE
Move MI CLI commands from apictl

### PR DESCRIPTION
**Pull request by chanikag**
This is to fix https://github.com/wso2/micro-integrator/issues/3387

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: cmd/utils/structs.go [hunk 1]
```diff
@@ -78,3 +78,40 @@ type KeyStore struct {
...
+// For main_config.yaml
+// To be manually edited by the user
+type MainConfig struct {
+	Config       Config                  `yaml:"config"`
+	Environments map[string]EnvEndpoints `yaml:"environments"`
+}
+
+type Config struct {
+	HttpRequestTimeout    int    `yaml:"http_request_timeout"`
+	ExportDirectory       string `yaml:"export_directory"`
```
Comment (SanojPunchihewa) [inline cmd/utils/structs.go:97]: Let's we remove these unused parameter holders.

**Comment by SanojPunchihewa** [comment]
Shall we move the MI related tests as well from https://github.com/wso2/product-apim-tooling/tree/master/import-export-cli/mi/integration

**Comment by chanikag** [comment]
> Shall we move the MI related tests as well from https://github.com/wso2/product-apim-tooling/tree/master/import-export-cli/mi/integration

This is added now.

File: cmd/cmd/add/add.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+package add
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-mi-tooling/cmd/utils"
+)
+
+const AddCmdLiteral = "add"
+const addCmdShortDesc = "Add new users or loggers to a Micro Integrator instance"
```
Comment (SanojPunchihewa) [inline cmd/cmd/add/add.go:27]:
```suggestion
const addCmdShortDesc = "Add new users, roles or loggers to a Micro Integrator instance"
```

Update the log description as well.

File: cmd/cmd/delete/delete.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+package delete
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-mi-tooling/cmd/utils"
+)
+
+const deleteCmdLiteral = "delete"
+const deleteCmdShortDesc = "Delete users from a Micro Integrator instance"
```
Comment (SanojPunchihewa) [inline cmd/cmd/delete/delete.go:27]:
```suggestion
const deleteCmdShortDesc = "Delete users or roles from a Micro Integrator instance"
```

File: cmd/cmd/root.go [hunk 4]
```diff
@@ -19,54 +19,110 @@
...
+func createConfigFiles() {
+	err := utils.CreateDirIfNotExist(utils.ConfigDirPath)
+	if err != nil {
+		utils.HandleErrorAndExit("Error creating config directory: "+utils.ConfigDirPath, err)
+	}
+
+	err = utils.CreateDirIfNotExist(utils.DefaultExportDirPath)
+	if err != nil {
+		utils.HandleErrorAndExit("Error creating config directory: "+utils.DefaultExportDirPath, err)
+	}
```
Comment (SanojPunchihewa) [inline cmd/cmd/root.go:91]: We can remove this exported directory as well. You may need to remove it from the `MainConfig` as well.

**Comment by SanojPunchihewa** [comment]
Let's remove the properties [1] from MessageStoreUtils[2] as well

[1] - https://github.com/wso2/product-apim-tooling/pull/892
[2] - https://github.com/wso2/product-mi-tooling/blob/master/cmd/utils/artifactUtils/messageStoreUtils.go

**Review by SanojPunchihewa — APPROVED** [review]

**Pull request merged**